contrib/gcppubsub/pkg/reconciler/gcppubsubsource_test.go

```diff
@@ -340,6 +384,35 @@ func getSource() *sourcesv1alpha1.GcpPubSubSource {
 	return obj
 }
 
+func getEventType() *eventingv1alpha1.EventType {
```

Comment:
this should be a method called `NewEventType` in the `./resources` directory for this reconciler.

Reply:
Oh, I am sorry, this is in the test.

IGNORE!